docs/using-data.md

```diff
@@ -100,6 +100,21 @@ would map to these coordinates
 github.com%2fsatori/go.uuid/v1.2.1-0.20181028125025-b2ce2384e17b
 ```
 
+Alternately, if you are using a component that has more than five slashes in it's namespace, you can use 
+an alternative version of the API to retrieve a definition for a coordinate (this is something we
+expect to expand in the future). This alterative API allows you to specify the coordinates as URL 
+query parameters.
+
```

Comment:
This version of the API involves slightly different encoding than the previous one (and it won't be clear to the user how it is different unless we say so specifically.

I suggest we add this to the documentation:

For a go component with an import url of `github.com/quasilyte/regex/syntax v0.0.0-20200419152657-af9db7f4a3ab`, the coordinates would look like this:

* type: go
* provider: golang
* namespace: quasilyte/regex (we will need to do some url encoding to this, see below)
* name: syntax
* revision: v0.0.0-20200419152657-af9db7f4a3ab

With this API, we need to encode the slashes in the namespace as '%252F'. So, for this example, the namespace we use in the coordinates is `github.com%252Fquasilyte%252Fregex`.  Our coordinates would now look like this:

* type: go
* provider: golang
* namespace: github.com%252Fquasilyte%252Fregex
* name: syntax
* revision: v0.0.0-20200419152657-af9db7f4a3ab

When using the `coordinates` parameter, we also need to encode the slashes in the coordinates. Put together, our coordinates look like this:

`go%2Fgolang%2Fgithub.com%252Fquasilyte%252Fregex%2Fsyntax%2Fv0.0.0`